model/bindings.go

```diff
@@ -0,0 +1,130 @@
+package model
+
+import (
+	"fmt"
+	log "github.com/Sirupsen/logrus"
+	yaml "github.com/cloudfoundry-incubator/candiedyaml"
+	"github.com/docker/libcompose/config"
+	"github.com/docker/libcompose/utils"
+	"github.com/rancher/rancher-compose/preprocess"
+	"github.com/rancher/rancher-compose/rancher"
+	"io/ioutil"
+	"os"
+	"reflect"
+)
+
+type mapLabel map[string]string
+
+//BindingProperty holds bindings
+type BindingProperty map[string]interface{}
+
+//ServiceBinding holds the fields for ServiceBinding
+type ServiceBinding struct {
+	Scale  int               `json:"scale"`
+	Labels map[string]string `json:"labels"`
+	Ports  []string          `json:"ports"`
+}
+
+//CreateBindingsRancher creates bindings property for rancher-compose.yml
+func CreateBindingsRancher(pathToYml string) BindingProperty {
+	var rawConfigRancher config.RawServiceMap
+	var rancherComposeMap map[string]rancher.RancherConfig
+	var bindingsMap map[string]ServiceBinding
+	// BindingProperty = make(map[string]interface{})
+	var bindingPropertyMap BindingProperty
+
+	var configMap map[string]config.ServiceConfigV1
+	var rawConfigDocker config.RawServiceMap
+
+	bindingsMap = make(map[string]ServiceBinding)
+	bindingPropertyMap = make(map[string]interface{})
+
+	rancherFile := pathToYml + "/rancher-compose.yml"
+
+	dockerFile := pathToYml + "/docker-compose.yml"
+	fileExists, err := os.Stat(dockerFile)
+	_ = fileExists
```

Comment:
These `_ = var` lines don't do anything (anything useful, anyway.. I know it fixes a warning in your IDE or something but need to figure out a real solution that isn't dead code)